bin/dev/versions.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+# Versions of the pieces pulled into SCF (outside of submodules).
+# This file is sourced everywhere one or more of the versions are
+# needed.
+
+# I. bin/dev/install_tool.sh
```

Comment:
I generally dislike numbered comment when the sequence isn't actually that important. It becomes a nuisance to renumber things when you insert something in the middle.

Also, roman numbers?